lib/app.rb

```diff
@@ -238,7 +238,22 @@ def redirect_via_js(path)
     end
 
     def supports_js_redirection?
-      browser.modern? && !memoized_cookies.empty?
+      modern_browser?(browser) && !memoized_cookies.empty?
+    end
```

Comment:
This check can be dropped completely, as "DevDocs being a developer tool, the browser requirements are high" (citation from README).

Reply:
@simon04 Double checking: we don't need just the `modern_browser?` check within this method or are you saying that the whole `supports_js_redirection?` method can be dropped?